jquery.flot.navigate.js

```diff
@@ -82,6 +82,18 @@ can set the default in the options.
             active: false,
             cursor: "move",
             frameRate: 60
+        },
+        xaxis: {
+            axisZoom: true, //zoom axis when mouse over it is allowed
+            plotZoom: true, //zoom axis is allowed for plot zoom
+            axisPan: true, //pan axis when mouse over it is allowed
+            plotPan: true //pan axis is allowed for plot pan
+        },
+        yaxis: {
+            axisZoom: true, //zoom axis when mouse over it is allowed
+            plotZoom: true, //zoom axis is allowed for plot zoom
+            axisPan: true, //pan axis when mouse over it is allowed
+            plotPan: true //pan axis is allowed for plot pan
```

Comment:
In case there are more than one y or more than one x axes, are these settings set to true for all of them?

Reply:
Yes, they are.